test/electron/fixtures/app/main.js

```diff
@@ -66,6 +69,11 @@ ipcMain.on('main-process-console-log', (_event, ...args) => {
   console.log(...args)
 })
 
+ipcMain.on('child-process-crash', () => {
+  const proc = fork(join(__dirname, 'sample-node-crash.js'))
+  proc.on('exit', (code, signal) => console.log(`child exit! ${code} - ${signal}`))
+})
```

Comment:
I guess this is leftover from trying to test child process crashes — do we want to keep it?